data/events/recurring.yml

```
@@ -2,60 +2,69 @@ recurring:
   - title: "OCaml Users in Paris (OUPS)"
     slug: "ocaml-users-paris-oups"
     event_type: meetup
-    textual_location: Paris, France
+    country: "France"
+    city: "Paris"
     location:
       lat: 48.8566
       long: 2.3522
     url: https://www.meetup.com/ocaml-paris/
   - title: "NYC OCaml Meetup"
     slug: "nyc-ocaml-meetup"
     event_type: meetup
-    textual_location: New York City, New York, USA
+    country: "USA"
+    city: "New York City"
     location:
       lat: 40.7128
       long: 74.0060
     url: https://www.meetup.com/NYC-OCaml
   - title: "Silicon Valley OCaml Meetup"
     slug: "silicon-valley-ocaml-meetup"
     event_type: meetup
-    textual_location: San Francisco, California, USA
+    country: "USA"
+    city: "San Francisco"
     location:
       lat: 37.7749
       long: 122.4194
     url: https://www.meetup.com/sv-ocaml/
   - title: "Cambridge NonDysFunctional Programmers"
     slug: "cambridge-nondysfunctional-programmers"
     event_type: meetup
-    textual_location: Cambridge, United Kingdom
+    country: "United Kingdom"
+    city: "Cambridge"
     location:
       lat: 52.2053
       long: 0.1218 
     url: https://www.meetup.com/Cambridge-NonDysFunctional-Programmers/
   - title: "MirageOS hack retreat"
```

```suggestion
  - title: "MirageOS Hack Retreat"
```
to match capitalisation in other titles